packages/tools/devtools/devtools-browser-extension/src/devtools/Logging.ts

```diff
@@ -23,3 +24,18 @@ export const devtoolsScriptMessageLoggingOptions: MessageLoggingOptions = {
 export function formatDevtoolsScriptMessageForLogging(text: string): string {
 	return `${devtoolsScriptLoggingContext}: ${text}`;
 }
+
+/**
+ * Logger that can be passed to the extension via a {@link @fluid-experimental/devtools-view#LoggerContext} to display
+ * usage telemetry events from the extension with alert popups in the browser.
+ *
+ * @remarks
+ * Only intended for testing the extension during development. Since the extension runs in a separate context than the
+ * current browser tab, console.log() does not display the messages anywhere we can use them for troubleshooting.
```

Comment:
Might be a silly question but, can't we also right-click in the extension to inspect console and get a new console window for our extension in which console logs should appear?

Reply:
Yes we can! TIL how to look at that console.